pkg/sbom/document.go

```diff
@@ -0,0 +1,70 @@
+package sbom
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type StandardType uint64
+
+const (
+	StandardTypeNone StandardType = iota
+	StandardTypeSpdx
+)
+
+func (t StandardType) String() string {
+	switch t {
+	case StandardTypeNone:
+		return "none"
+	case StandardTypeSpdx:
+		return "spdx"
+	default:
+		panic("invalid standard type")
+	}
+}
+
+func (t StandardType) MarshalJSON() ([]byte, error) {
+	var s string
+
+	if t == StandardTypeNone {
+		s = ""
```

Comment:
Is the string `"none"` being used anywhere? Because maybe we could change the `String()` method to just return the empty string and this `if/else` wouldn't be necessary.

I may be missing something though

Reply:
It is used if you would print the value as string. I wanted to keep there something that's descriptive. For JSON, I want the value to be omitted if empty (being "none") and representing it as an empty string make things easier to work with the standard JSON (un)marshaller.

Reply:
Cool thanks, that seems sensible